sql/catalyst/src/test/scala/org/apache/spark/sql/connector/catalog/InMemoryTableCatalog.scala

```diff
@@ -268,6 +268,11 @@ class InMemoryTableCatalog extends BasicInMemoryTableCatalog with SupportsNamesp
     procedure
   }
 
+  override def listProcedures(namespace: Array[String]): Array[Identifier] = {
+    procedures.asScala.filter{case (_, p) => !p.name().equals("dummy_increment")}
```

Comment:
```suggestion
    procedures.filter{case (_, p) => !p.name().equals("dummy_increment")}
```

Reply:
i found an implicit conversion 'scala.collection.convert.ImplicitConversions.`map AsScala`' but it seems deprecated for Scala 2.13.  Did you have another implicit in mind?

Reply:
oh nvm, I misread the code.

BTW it's scala s we can do `case (_, p) if p.mame != "..."`

Reply:
this is filter, it has to return predicate, do you mean       ```.filter{case (_, p) if !p.name().equals("dummy_increment") => true}```?  it looks a bit weird.